config/settings.yml

```diff
@@ -1444,10 +1444,21 @@ genisis:
   pass: bogus
 
 # Settings for connecting AFS Veteran Services
+# For locahost we can use the existing certs as long as we don't call out
 dgi:
+  # add med_api here? Will need to reach out to that team
   jwt:
-    public_key_path: modules/meb_api/spec/fixtures/dgi_public_test.pem
-    private_key_path: modules/meb_api/spec/fixtures/dgi_private_test.pem
+    public_key_path: ""
```

Comment:
Any reason you didn't just keep the path to the fixture here, also why not use it for `vye.jwt.public_key_path`, and `vye.jwt.private_key_path`? 

I think you should expect this to hard crash in `lib/vye/dgib/authentication_token_service.rb` and `modules/meb_api/lib/authentication_token_service.rb` if there isn't a valid certificate/key path. Otherwise, what will happen on prod, if the cert/key file doesn't deploy for whatever reason, the app will create tokens without a signature. I think crashing is vastly preferred to that

Reply:
These paths have been added back in. Thanks.